core/storage/topic/gae_models.py

```diff
@@ -241,6 +241,8 @@ class TopicSummaryModel(base_models.BaseModel):
     canonical_name = ndb.StringProperty(required=True, indexed=True)
     # The ISO 639-1 code for the language this topic is written in.
     language_code = ndb.StringProperty(required=True, indexed=True)
+    # The description of the topic.
+    description = ndb.TextProperty(indexed=False)
```

Comment:
This would require a migration job, since the topic summary models in the datastore now wouldn't have this field populated. So, can you add that as well?

Reply:
For summary models, we should regenerate them right? I.e. it's not so much a migration as a regeneration from scratch, since the summaries have no schema version...

Reply:
Yeah, it's more like overwriting existing data with new ones (which does open if we normally update the topic manually).

Reply:
Yeah, I think we'll need to run a job. Thankfully, it should be quick since there aren't many topics at the moment.